.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Bug report
+about: Create a bug report
+---
+
+<!--
+
+Do you want to ask a question? Are you looking for support? GitHub Discussions is the best place for getting support.
+
+-->
+
+* [ ] I've read the guidelines for [Contributing](https://github.com/area17/twill/blob/master/CONTRIBUTING.md)
+* [ ] This request isn't a duplicate of an existing issue
+* [ ] I've read the [docs](https://twill.io/docs/) and followed them (if applicable)
+* [ ] This is not a personal support request that should be posted on [GitHub Discussions](https://github.com/area17/twill/discussions) or [Twill Discord]()
+
+## Description
+
+Describe the issue that you're seeing.
+
+### Steps to reproduce
+
+Clear steps describing how to reproduce the issue.
+
+### Expected result
+
+What should happen?
+
+### Actual result
+
+What happened.
+
+### Versions
+
+What Twill version are you using? Also, please include the OS and what version of the OS you're running.
```

Comment:
I think will be good ask about the database engine being used (MySQL/MariaDB, PostgreSQL, SQL Server, etc), so possible query issues can be investigated.